File: 0201_0300/0233/0233.py
with open("input.txt", "r") as f, open("output.txt", "w") as q:
    n = int(f.readline())
    s = [int(x) for x in f.read().split()]
    k = next((i for i in range(n) if s[i] <= 437), -1)
    q.write("No crash" if k < 0 else "Crash {}".format(k + 1))

File: 0201_0300/0233/0233.c
#include <stdio.h>

int main()
{
	FILE *f = fopen("input.txt", "r"), *q = fopen("output.txt", "w");
	int n, k, i;
	fscanf(f, "%d", &n);
	for (i = 1; i <= n; ++i)
	{
		fscanf(f, "%d", &k);
		if (k <= 437)
			break;
	}
	fprintf(q, i > n ? "No crash" : "Crash %d", i);
	return 0;
}
